src/mods.h

```diff
@@ -45,16 +45,22 @@ struct ModSpec
 	bool is_modpack = false;
 	// if modpack:
 	std::map<std::string,ModSpec> modpack_content;
-	ModSpec(const std::string &name_="", const std::string &path_=""):
+	ModSpec(const std::string &name_ = "", const std::string &path_ = ""):
 		name(name_),
 		path(path_)
 	{}
+	ModSpec(const std::string &name_, const std::string &path_, bool p_o_m):
+		name(name_),
+		path(path_),
+		part_of_modpack(p_o_m)
```

Comment:
`part_of_modpack_` for consistency and the IDEs that provide information from the headers.